xml/System.Web.Configuration/HttpModuleAction.xml

```diff
@@ -31,7 +31,7 @@
   
  The following configuration file example shows how to specify values declaratively for the `httpModules` section.  
   
- Refer to the [NIB: How to: Create and Configure an HTTP Module](https://msdn.microsoft.com/library/4174fd13-a9c4-4ff6-baa5-3f7d7fc73ef3) to see how to build and configure an HTTP module.  
+ Refer to the [How to: Create and Configure an HTTP Module](/previous-versions/dotnet/netframework-3.0/tfd6k449(v=vs.85)) to see how to build and configure an HTTP module.  
```

Comment:
You can use the full URL for these. They wouldn't work in the offline books.

```suggestion
 Refer to the [How to: Create and Configure an HTTP Module](https://docs.microsoft.com/previous-versions/dotnet/netframework-3.0/tfd6k449(v=vs.85)) to see how to build and configure an HTTP module.  
```

Reply:
Ok I'll change the others also.
Does this also apply to the [dotnet/docs](https://github.com/dotnet/docs) repository?

Reply:
for previous-versions links, yes

Reply:
I think I've forgotten this rule recently... I may have to check one or two recent commits on the docs repo!

Reply:
It's not a big deal. Don't worry!